src/components/forms/UpdateProfile.js

```diff
@@ -0,0 +1,198 @@
+import React from "react";
+import { bindActionCreators } from "redux";
+import { connect } from "react-redux";
+import { TextField, withStyles, Button } from "@material-ui/core";
+import { Formik } from "formik";
+
+import { putNewProfileInfo } from "Redux/state/profile/actions";
+import { updateProfile } from "Services/Auth/RegistrationApi";
+import CenterPaper from "components/layout/CenterPaper";
+
+const styles = () => ({
+  picture: {
+    width: "128px",
+    height: "128px",
+    borderRadius: "50%",
+    border: "3px solid black",
+  },
+});
+
+const txtfields = {
+  width: "100%",
+  marginTop: "25px",
+};
+
+function UpdateProfile(props) {
+  const { classes, reduxState, profileUpdate } = props;
+  const {
+    first_name, last_name, birth_date, phone_number, id, user,
+  } = reduxState.profile;
+
+  // const userGender = [
+  //   {
+  //     value: "male",
+  //     label: "Male",
+  //   },
+  //   {
+  //     value: "female",
+  //     label: "Female",
+  //   },
+  // ];
+
+  return (
+    <div>
+      <hr />
+      <CenterPaper>
+        <Formik
+          initialValues={{
+            fname: first_name,
+            lname: last_name,
+            birthdate: birth_date,
+            phonenum: phone_number,
+            // gender: reduxState.profile.gender,
+          }}
+          onSubmit={async (values) => {
+            const {
+              fname, lname, birthdate, phonenum,
+              // gender,
+            } = values;
+
+            // let avatarUrl = null;
```

Comment:
don't leave comment code only if you need it in the future, otherwise please remove